### PR TITLE
sv employee side UI added

### PR DESCRIPTION
**Pull request by rashidkhalid1212**
1. SV Employee Side Card Added
2. Employee Side UI Forms Added using Same Component and Config File
3. Employee Side Parent Component Added so that it will not conflict with Citizen side
4. Index Routing Page to managethe routing of EMployee SIde  
5. Employee Side Check page added

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nudmcdgnpm/upyog-ui-module-ptr",
-  "author": "Shivank Shukla & Abhijeet Verma",
+  "author": "Shivank Shukla, Abhijeet Verma & Khalid Rashid",
```
Comment (ShivankShuklaa) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/package.json:3]: remove this

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/Module.js [hunk 2]
```diff
@@ -86,9 +120,15 @@ export const PTRModule = ({ stateCode, userType, tenants }) => {
     []
   );
 
-  if (userType === "employee") {
-    return <EmployeeApp path={path} url={url} userType={userType} />;
-  } else return <CitizenApp />;
+  return ( //application context provider to pass context data to all child components
+    <ApplicationContext.Provider value={{ applicationId, setApplicationId, applicationData, apptype, setApptype}}>
```
Comment (rashidkhalid1212) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/Module.js:124]: refactor code to get the data from state first then set the data in context

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/Module.js [hunk 3]
```diff
@@ -100,17 +140,17 @@ export const PTRLinks = ({ matchPath, userType }) => {
   }, []);
 
   const links = [
```
Comment (rashidkhalid1212) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/Module.js:142]: remove this code

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/config/Create/citizenconfig.js [hunk 4]
```diff
@@ -1,115 +1,129 @@
...
                         "isMandatory": true,
                         "texts": {
-                            "submitBarLabel": "PTR_COMMON_NEXT",
+                            "submitBarLabel": "COMMON_NEXT",
+                            "header": "PROPERTY_DETAILS_HEADER",
                         }
-                        },
-                    ],
                     },
+                    // {
```
Comment (rashidkhalid1212) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/config/Create/citizenconfig.js:67]: use multiline comments

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenDetails.js [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
 
 const PTRCitizenDetails
  = ({ t, config, onSelect, userType, formData, ownerIndex }) => {
   const { pathname: url } = useLocation();
 
   let index = 0
-  
-   
+
+  const {applicationData} = useContext(ApplicationContext)
```
Comment (rashidkhalid1212) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenDetails.js:14]: add comment

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenDetails.js [hunk 6]
```diff
@@ -32,6 +33,15 @@ const PTRCitizenDetails
   const tenantId = Digit.ULBService.getCurrentTenantId();
   const stateId = Digit.ULBService.getStateId();
 
+  // useeffect used to fill the previous application details 
+  useEffect(() => {
+    setName(applicationData?.applicantName)
```
Comment (rashidkhalid1212) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenDetails.js:38]: check this once if we can use state

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenPet.js [hunk 7]
```diff
@@ -41,20 +44,40 @@ const PTRCitizenPet
...
 
-    const { data: Breed_Type } = Digit.Hooks.ptr.useBreedTypeMDMS(stateId, "PetService", "BreedType");  // hooks for breed type
+    const { data: Menu, isLoading: pettype_load } = Digit.Hooks.ptr.usePTRPetMDMS(stateId, "PetService", "PetType");
+
+    const { data: Breed_Type, isLoading: Loading_breed } = Digit.Hooks.ptr.useBreedTypeMDMS(stateId, "PetService", "BreedType");  // hooks for breed type
 
     let menu = [];   //variable name for pettype
-    let breed_type = [];
-    // variable name for breedtype
+    let breed_type = [];    // variable name for breedtype
```
Comment (rashidkhalid1212) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenPet.js:80]: add proper comments

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenPet.js [hunk 8]
```diff
@@ -131,19 +197,28 @@ const PTRCitizenPet
     }
 
 
+    useEffect(() => {
+      if (birthDate) {
+        setAdoptionDate(null);
+      }
+    }, [birthDate])
```
Comment (rashidkhalid1212) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/ptr/src/pageComponents/PTRCitizenPet.js:205]: add semi column

**Review by rashidkhalid1212 — COMMENTED** [review]
please Check

**Pull request merged**